screeninfo/_drm.py

```diff
@@ -0,0 +1,230 @@
+"""
+Author: Marcin Kurczewski and others
+License: MIT License
+Copyright (C) 2015 Marcin Kurczewski
+"""
+import os
+from ._monitor import Monitor
+from ._utils import load_library
+
+
+def enumerate_drm():
+    """Create a list of Monitor instances on the ChromeOS platform."""
```

Comment:
That's not true. DRM is used by Linux kernel, ChromeOS is based on that.
Roughly speaking, DRM is the graphical system you see before you run X.

Reply:
I did not know that, thank you. I will edit the docstring accordingly.